### ISSUE TITLE
Multiplayer: Lights only correctly show on/off when local planes lights/power is enabled

### ISSUE THREAD

**Issue by hbeni** [HHS81/c182s#418]
Remote C182 light is only showed when the local electircal bus has power (and also the lights are switched on; but this was not tested dedicately; so I currently assume it's a power-only thing).

Probably the electrical system property is not correctly relayed and depends on a local value?

**Comment by hbeni** [comment]
I could not reproduce this, when the local plane is _not_ an C182.
Lights work normal and as expected when viewed from the C172P and PA-28

**Comment by hbeni** [comment]
For the icing, I maybe have found a problematic code piece:
https://github.com/HHS81/c182s/blob/ed9bb50ae9741d11b84c9a44aa98a84022804fc8/Nasal/c182s.nas#L512-L522

**Comment by hbeni** [comment]
Hm, the lights problem did not appear in the [last session](https://forum.flightgear.org/viewtopic.php?p=392467#p392467):

https://i.ibb.co/ZB8F2sm/fgfs-20210922204625.jpg

**Comment by TheFGFSEagle** [comment]
Yes - if there is a property named "dummy" in /ai/models/multiplayer[n], that will be used - if it's not there, FG will look for it in the global property tree of the remote instance.

**Comment by hbeni** [comment]
So, this is a shader issue and cannot be fixed plane sided?

**Comment by TheFGFSEagle** [comment]
Uh, it can be fixed plane side - just ensure that all properties the shader should use the remote value of are transmitted over MP ! :)

**Comment by hbeni** [comment]
> Uh, it can be fixed plane side - just ensure that all properties the shader should use the remote value of are transmitted over MP ! :)

That is already the Case. All is working fine when you look at iced remote c182s when you are self _not_ in one.
As soon as you are in a 182 yourself, both your and the remote instances always shows the icing of your own plane.

**Comment by TheFGFSEagle** [comment]
hum, that is weird … could you check on another aircraft maybe ?

**Comment by hbeni** [comment]
What do you mean, what exactly do you echt to have tested?

**Comment by TheFGFSEagle** [comment]
I want to know if you can observe the same behaviour replacing the 182 with another aircraft (172 or J-3 Cub for example) - if the issue happens there too, something must be wrong within core FG … but generally I don't know whatever could be going on here. It looks to me like the global propery takes precedence over the one in /ai/models/aircraft[n] which should not happen …

**Comment by hbeni** [comment]
So we need to test this:
„On a plane, that implements icing graphics, does it work on multiplayer if there are is just that model around?“

Seneca-II, or maybe the c172 when it git ready.

**Comment by hbeni** [comment]
A tought completely the same Butt expressed differently:
It looks like the shader can only have one singular effect, and when a local effect is in place, it is applied to all available instances, regardless if they should have their own, distinct effect.

Do we maybe have some bottleneck in the effects definition, like we do currently have with the glass/rain, and also the lights?

**Comment by hbeni** [comment]
After testing https://github.com/HHS81/c182s/issues/518 I could not reproduce the lighting issue anymore -> looks fixed.

Icing display on the remote plane still is broken when local plane is an C182s/t (just tested).
